dist/_patterns/03-templates/03-article-2col.twig

```diff
@@ -1,4 +1,4 @@
-{% extends "@templates/page-2col.twig" %}
+{% extends "@templates/00-layouts/01-page-2col.twig" %}
```

Comment:
This reference is just incorrect.

Other `include`s within the template appear to be incorrect as well, but they do not prevent the generation from completing. May want to consider updating all of these, possibly using the [pattern include partial syntax](https://github.com/pattern-lab/patternengine-php-twig#pattern-includes).